bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/forms2/contract/form/FormType.java

```diff
@@ -0,0 +1,16 @@
+package org.bahmni.module.bahmnicore.forms2.contract.form;
```

Comment:
Since this has been moved to a "forms2" specific package, does it still require a package "contact.form"? it can be just "contract". (personally I don't like the name contract, but ok)